@css/main_style.css

```diff
@@ -691,8 +691,11 @@ dl.tech-menu > dt {
 }
 
 .form-name-layout {
+  display: flex;
   min-width: 327px;
-  height: 48px;
+  height: 18px;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
- [ ] According to the project requirement I would kindly suggest you use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels for proper accessibility,

Reply:
The  reason why the values are in px and not in percentages is due the fact that in the Figma template they are expressed in px values.

And from start I did not made any conversions f or relative values -- started using global variables, to hold those values, for easy change in future, wherever needed -- and now it will take some time I don't know I have to go through all of them and change them to percentages and ensure it all work without any side effects, in the limited amount of time I have.

Reply:
Another thing that might have had some influence is the fact that the online site where this commit was available was the same where I was using to continue the work and do the online testing I always do, while developing.

That issue is already solved, as this (at this branch) has now it's own directory, so it wont be affected by the work sill being done on this project.